backend/tests/conftest.py

```diff
@@ -32,21 +47,33 @@ def test_project_name():
 
 
 @pytest.fixture()
-def logged_in_client(test_client) -> str:
-    response = test_client.post(f"{API_VERSION_PREFIX}/users")
-    test_client.cookies = response.cookies
-    return test_client
+def logged_in_client(mock_client, mock_user_id) -> str:
+    cookie = {"user_id": str(mock_user_id)}
+    mock_client.cookies = cookie
+    return mock_client
 
 
 @pytest.fixture()
-def created_project_id(logged_in_client, test_project_name):
-    data = {"name": test_project_name}
-    response = logged_in_client.post(f"{API_VERSION_PREFIX}/projects", json=data)
-    json_result = response.json()
-    return json_result.get("id")
+def created_project_id(test_project_name, mock_user_id):
```

Comment:
`project_id` seems fine as this fixture's name. Use prefix to differentiate. here, it's the normal, expected, behavior.
Mind the scope as well.

Reply:
The `project_id` should also be in the `function` scope. Each test should have its own project to avoid side effects.